pages/index.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Abby Choi</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <link href="/styles/index.css" rel="stylesheet">
+  
 </head>
+
 <body>
-  
+<div class="container">
+<header>
+  <h1>Abby Choi</h1>
+  <nav>
+    <ul>
+      <li><a id="index" href="/pages/index.html">Home</a></li>
+      <li><a href="/pages/about.html">About Me</a></li>
+      <li><a href="/pages/portfolio.html">Portfolio</a></li>
+    </ul>
+  </nav>
+</header>
+
+<main>
+  <p>
+  Hi, it's nice to meet you! Welcome to a visualization of my humble beginnings. There isn't much to see yet, but if you come along for the ride, you can watch over my growth :)
+  </p>
+</main>
+
+<footer>
+  <h3>Let's Connect!</h3>
+    <nav>
+    <a href="abbychoi@gmail.com"><img class="icon" src="/assets/icons8-mail-96.png" alt="Email Icon"></a>
```

Comment:
To make this link to an interface where someone can send an email, add `mailto:` in front of your email:

`<a href="mailto:abbychoi@gmail.com">...</a>`